plugins/inventory_sync/module/define_player_stat_keys.lua

```diff
@@ -14,10 +14,4 @@ return {
 	"character_maximum_following_robot_count_bonus",
 	"character_health_bonus",
 	"character_personal_logistic_requests_enabled",
-	"build_distance",
```

Comment:
wh not sync these any more? Admins can modify the stats of the users via lua, so they might be different than calculated. Also, RPG mods? Or is this somehow synced another way?

Reply:
If you read the docs the reason is obvious [these properties are read only](https://lua-api.factorio.com/latest/LuaControl.html#LuaControl.build_distance), you cannot set them, and trying to do so throws an error.